tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -273,9 +273,7 @@ def _SetupWSGIApp(self):
     context = base_plugin.TBContext(
         logdir=self.log_dir, multiplexer=multiplexer)
     self.plugin = projector_plugin.ProjectorPlugin(context)
-    wsgi_app = application.TensorBoardWSGIApp(
-        self.log_dir, [self.plugin], multiplexer, reload_interval=0,
```

Comment:
This had `reload_interval=0`, so if I understand correctly we would
usually need to `AddRunsFromDirectory` and `Reload` manually here to
preserve existing behavior. Is it just that the projector plugin doesn’t
actually query data from the multiplexer except for `PluginAssets`, and
the plugin assets data path is not covered by these tests?

Reply:
Yes, I think that's basically the case. Specifically the tests here just rely on the fallback behavior that checks the root logdir directly (in the case of no event files at all):
https://github.com/tensorflow/tensorboard/blob/15a63688e6e711f80aa62811bcb3761d02be8b3b/tensorboard/plugins/projector/projector_plugin.py#L331-L334

Thanks for pointing this out though since it reminded me I was going to file a bug based on what I determined after looking at this part of the mesh plugin test.